Client/src/Components/CSS/NewsAPI.css

```diff
@@ -6,7 +6,7 @@
 }
 
 .article-container {
-    background: rgb(241, 235, 241);
+    background: aliceblue;
```

Comment:
Nit: styling changes should be committed in a different branch since this PR is focused on testing

Reply:
I couldn't help myself and took a jab at CSS but mostly focused on testing.

Reply:
🤣